controllers/integration/integration_adapter.go

```diff
@@ -360,6 +360,32 @@ func (a *Adapter) createMissingReleasesForReleaseLinks(releaseLinks *[]releasev1
 	return nil
 }
 
+// getRequiredIntegrationTestScenariosForApplication returns the ReleaseLinks used by the application being processed. If matching
```

Comment:
nit: update the doc line to reflect the new function

Reply:
Thanks for pointing that out, updated.